### PR TITLE
Handle double-quoted headers

### PR DESCRIPTION
**Pull request by martonpe**

### CODE REVIEW

**Comment by gszjulcsi** [comment]
Don't you mean quoted headers? ;)

**Comment by martonpe** [comment]
@gszjulcsi :hankey:, I do :)

File: spec/support/data_parenthesized.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"id","name","description"
+1,dresses,Dresses
+"2","female-tops",
+"3",,"Bras"
```
Comment (gszjulcsi) [inline spec/support/data_parenthesized.csv]: why is this a bad row when the next few line are ok?

Comment (gszjulcsi) [inline spec/support/data_parenthesized.csv]: Oh, I see, because name is missing!

**Comment by gszjulcsi** [comment]
LGTM with fixed naming

**Comment by martonpe** [comment]
Closing in favor of https://github.com/sspinc/csv2avro/pull/27 with correct naming.

**Pull request closed without merging**